packages/next-swc/crates/next-api/src/app.rs

```diff
@@ -672,6 +683,67 @@ impl AppEndpoint {
             )))
         }
 
+        async fn create_react_loadable_manifest(
```

Comment:
Can we extract this and share with the pages impl?

Reply:
I recall there was slight differences between those and decided to go with dupe for now.